CHANGELOG.md

```diff
@@ -1,3 +1,80 @@
+- [fluentbit] Reduced the default FluentBit logging level to warning.
+- [mbed-fcce] Upgrades factory-configurator-client-example to v4.9.0.
+- No longer required to specify this `vendor-id=42fa7b48-1a65-43aa-890f-8c704daade54` to unlock the rich node features in Pelion web portal. Portal now reads the Gateway capabilities from gateway's FeatureMgmt LwM2M object 33457 and then enable/disable the UI associated to the feature.
+
+### Bug fixes
+
+- [info] The `info` command must be run with `sudo` on LMP-based boards (UltraZed-EG IOCC and i.MX 8M Mini EVK).
+- [info] The `info` command on the UltraZed-EG IOCC attempts to read the CPU temperature when the temperature file does not exist. This results in a cat error message.
+- The LmP updates don't accept firmware updates with numbers 10 and 100. By default, the tenth update you perform is version 0.0.10, which fails. To work around this problem, avoid software version numbers with 10 and 100:
+   - For example, you can go from 0.0.9 to 0.0.11 by editing the `MAIN` component version information in a file called `~/Pelion_Edge_Credentials/.manifest-dev-tool$ cat update.version.yaml`. Change `MAIN: 0.0.10` to `MAIN: 0.0.11`.
+
+### Known issues
+
+- The Pelion Device Management portal is not correctly updated after a firmware campaign in some instances.
+- [maestro] The FeatureMgmt config resource is initialized with a maximum 3.8KB of file content. The remaining file content is truncated during initialization. This is most likely due to the limitation of the gorilla/websocket library but needs further investigation. However, you can still push a file size of a maximum of 64KB through cloud service APIs.
+- [pt-example] `cpu-temperature` device reports random values because the default CPU temperature file is not the same on Yocto and LmP.
+- The LmP build will enable SW TPM and Parsec stacks by default in all configuration, including developer certificate configurations. However, as it will not be used or set up in those configurations the logs will show some TPM related errors - those logs can be ignored.
+
+#### AVNET ZU3EG
+
+- If you enable kernel configurations [CPU_IDLE](https://cateee.net/lkddb/web-lkddb/CPU_IDLE.html) and [PREEMPT](https://cateee.net/lkddb/web-lkddb/PREEMPT.html), the LmP release including PetaLinux 2020.2 does not work in a stable manner. Our default configuration has those disabled. If you have any issues with those configurations, please contact Xilinx support.
+- You cannot do firmware update from Edge 2.2 to Edge 2.3 on the AVNET ZU3EG board due to LmP v79 release FPGA-support changes. The changes have interdepencies between the BOOT image and kernel image and as in the current update you can only update ther kernel image it fails to boot up correctly with the Edge 2.2 based BOOT image (as it does not supply the required updated device tree/FPGA files etc.). So, update to Edge 2.3 image must be done with manual flashing on ZU3EG targets.
```

Comment:
@ygoyal18 : This firmware update issue is across all the boards. cc: @petedyerarm

Reply:
I think this should be removed and due to to partition changes updates from 2.3 -> 2.4 are not possible, on any targets. Not just AVNET ZU3EG.

Reply:
However, it is still a critical issue. We should move it to the known issue list.

Reply:
Its not really an "issue." An issue is something that can be fixed.  My suggestion is we make it a limitation.  See my suggestion below.
```suggestion
```